Rakefile

```diff
@@ -57,24 +57,18 @@ end
 task :build, [:build_fat] do |t, args|
   args.with_defaults(:build_fat => false)
   print_info "Building project (fat: #{args[:build_fat]})"
-  sh %Q(swift build -c release --arch x86_64 --arch arm64 --disable-sandbox --build-path #{BUILD_DIR})
+
+  # Use xcodebuild due to https://bugs.swift.org/browse/SR-15802
+  xcpretty %Q(xcodebuild -resolvePackageDependencies)
+  xcpretty %Q(xcodebuild -scheme sourcery -destination platform=macOS,arch=x86_64 -derivedDataPath #{BUILD_DIR}derivedData -archivePath #{BUILD_DIR}sourcery.xcarchive archive)
+
+  # Prepare the export direcotry
   sh %Q(rm -fr #{CLI_DIR})
   sh %Q(mkdir -p "#{CLI_DIR}bin")
-  sh %Q(mkdir -p "#{CLI_DIR}lib")
-  
-if args[:build_fat] != true  # Build thin Library
-  ARCH = `uname -m`.chomp
-  sh %Q(lipo -thin #{ARCH} lib_InternalSwiftSyntaxParser.dylib -output #{CLI_DIR}lib/lib_InternalSwiftSyntaxParser.dylib)
-else
-  sh %Q(cp lib_InternalSwiftSyntaxParser.dylib #{CLI_DIR}lib)
-end
 
+  # Export the build products and clean up
   sh %Q(cp SourceryJS/Resources/ejs.js #{CLI_DIR}bin)
-  `mv #{BUILD_DIR}apple/Products/Release/sourcery #{CLI_DIR}bin/`
-  #`mv #{BUILD_DIR}release/Sourcery_SourceryJS.bundle #{CLI_DIR}lib/`
-  `install_name_tool -delete_rpath @loader_path #{CLI_DIR}bin/sourcery`
-  `install_name_tool -delete_rpath /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx #{CLI_DIR}bin/sourcery`
-  `install_name_tool -add_rpath "@executable_path/../lib" "#{CLI_DIR}bin/sourcery"`
```

Comment:
I think this is all redundant now. Here is the output from `otool -l`:

<details>
<summary><b>Output</b></summary>

```
cli/bin/sourcery (architecture x86_64):
Load command 0
      cmd LC_SEGMENT_64
  cmdsize 72
  segname __PAGEZERO
   vmaddr 0x0000000000000000
   vmsize 0x0000000100000000
  fileoff 0
 filesize 0
  maxprot 0x00000000
 initprot 0x00000000
   nsects 0
    flags 0x0
Load command 1
      cmd LC_SEGMENT_64
  cmdsize 1752
  segname __TEXT
   vmaddr 0x0000000100000000
   vmsize 0x0000000002574000
  fileoff 0
 filesize 39272448
  maxprot 0x00000005
 initprot 0x00000005
   nsects 21
    flags 0x0
Section
  sectname __text
   segname __TEXT
      addr 0x00000001000055e0
      size 0x00000000021c703a
    offset 21984
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x80000400
 reserved1 0
 reserved2 0
Section
  sectname __stubs
   segname __TEXT
      addr 0x00000001021cc61a
      size 0x0000000000001566
    offset 35440154
     align 2^1 (2)
    reloff 0
    nreloc 0
     flags 0x80000408
 reserved1 0 (index into indirect symbol table)
 reserved2 6 (size of stubs)
Section
  sectname __stub_helper
   segname __TEXT
      addr 0x00000001021cdb80
      size 0x0000000000002388
    offset 35445632
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x80000400
 reserved1 0
 reserved2 0
Section
  sectname __const
   segname __TEXT
      addr 0x00000001021cff10
      size 0x000000000013a2db
    offset 35454736
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_entry
   segname __TEXT
      addr 0x000000010230a1ec
      size 0x0000000000000004
    offset 36741612
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __cstring
   segname __TEXT
      addr 0x000000010230a1f0
      size 0x00000000001fcfbe
    offset 36741616
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __objc_methname
   segname __TEXT
      addr 0x00000001025071ae
      size 0x0000000000002a30
    offset 38826414
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __swift5_typeref
   segname __TEXT
      addr 0x0000000102509bde
      size 0x000000000000b832
    offset 38837214
     align 2^1 (2)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_reflstr
   segname __TEXT
      addr 0x0000000102515410
      size 0x0000000000008735
    offset 38884368
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_fieldmd
   segname __TEXT
      addr 0x000000010251db48
      size 0x000000000000cbe4
    offset 38918984
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_proto
   segname __TEXT
      addr 0x000000010252a72c
      size 0x0000000000003618
    offset 38971180
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_types
   segname __TEXT
      addr 0x000000010252dd44
      size 0x0000000000000fac
    offset 38985028
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_assocty
   segname __TEXT
      addr 0x000000010252ecf0
      size 0x0000000000002970
    offset 38989040
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_builtin
   segname __TEXT
      addr 0x0000000102531660
      size 0x0000000000000708
    offset 38999648
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_protos
   segname __TEXT
      addr 0x0000000102531d68
      size 0x00000000000000ec
    offset 39001448
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_capture
   segname __TEXT
      addr 0x0000000102531e54
      size 0x0000000000007284
    offset 39001684
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __gcc_except_tab
   segname __TEXT
      addr 0x00000001025390d8
      size 0x0000000000000044
    offset 39031000
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_classname
   segname __TEXT
      addr 0x000000010253911c
      size 0x000000000000000e
    offset 39031068
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __objc_methtype
   segname __TEXT
      addr 0x000000010253912a
      size 0x000000000000001f
    offset 39031082
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __unwind_info
   segname __TEXT
      addr 0x000000010253914c
      size 0x000000000002d9e4
    offset 39031116
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __eh_frame
   segname __TEXT
      addr 0x0000000102566b30
      size 0x000000000000d4d0
    offset 39217968
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Load command 2
      cmd LC_SEGMENT_64
  cmdsize 1752
  segname __DATA
   vmaddr 0x0000000102574000
   vmsize 0x0000000000168000
  fileoff 39272448
 filesize 1048576
  maxprot 0x00000003
 initprot 0x00000003
   nsects 21
    flags 0x0
Section
  sectname __nl_symbol_ptr
   segname __DATA
      addr 0x0000000102574000
      size 0x0000000000000008
    offset 39272448
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000006
 reserved1 913 (index into indirect symbol table)
 reserved2 0
Section
  sectname __got
   segname __DATA
      addr 0x0000000102574008
      size 0x0000000000003a70
    offset 39272456
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000006
 reserved1 914 (index into indirect symbol table)
 reserved2 0
Section
  sectname __la_symbol_ptr
   segname __DATA
      addr 0x0000000102577a78
      size 0x0000000000001c88
    offset 39287416
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000007
 reserved1 2784 (index into indirect symbol table)
 reserved2 0
Section
  sectname __mod_init_func
   segname __DATA
      addr 0x0000000102579700
      size 0x00000000000000e8
    offset 39294720
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000009
 reserved1 0
 reserved2 0
Section
  sectname __const
   segname __DATA
      addr 0x00000001025797f0
      size 0x00000000000a5da0
    offset 39294960
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_classlist
   segname __DATA
      addr 0x000000010261f590
      size 0x0000000000000650
    offset 39974288
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_catlist
   segname __DATA
      addr 0x000000010261fbe0
      size 0x0000000000000010
    offset 39975904
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_protolist
   segname __DATA
      addr 0x000000010261fbf0
      size 0x0000000000000118
    offset 39975920
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_imageinfo
   segname __DATA
      addr 0x000000010261fd08
      size 0x0000000000000008
    offset 39976200
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_const
   segname __DATA
      addr 0x000000010261fd10
      size 0x0000000000012a40
    offset 39976208
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_selrefs
   segname __DATA
      addr 0x0000000102632750
      size 0x0000000000000528
    offset 40052560
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000005
 reserved1 0
 reserved2 0
Section
  sectname __objc_protorefs
   segname __DATA
      addr 0x0000000102632c78
      size 0x0000000000000118
    offset 40053880
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_classrefs
   segname __DATA
      addr 0x0000000102632d90
      size 0x0000000000000118
    offset 40054160
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_data
   segname __DATA
      addr 0x0000000102632ea8
      size 0x000000000000ba98
    offset 40054440
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __data
   segname __DATA
      addr 0x000000010263e940
      size 0x000000000003247e
    offset 40102208
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __thread_vars
   segname __DATA
      addr 0x0000000102670dc0
      size 0x0000000000000060
    offset 40308160
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000013
 reserved1 0
 reserved2 0
Section
  sectname __swift_hooks
   segname __DATA
      addr 0x0000000102670e20
      size 0x00000000000000b8
    offset 40308256
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift51_hooks
   segname __DATA
      addr 0x0000000102670ed8
      size 0x00000000000000b8
    offset 40308440
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __thread_bss
   segname __DATA
      addr 0x0000000102670f90
      size 0x0000000000000020
    offset 0
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000012
 reserved1 0
 reserved2 0
Section
  sectname __bss
   segname __DATA
      addr 0x0000000102670fb0
      size 0x0000000000067548
    offset 0
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000001
 reserved1 0
 reserved2 0
Section
  sectname __common
   segname __DATA
      addr 0x00000001026d8500
      size 0x0000000000000719
    offset 0
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000001
 reserved1 0
 reserved2 0
Load command 3
      cmd LC_SEGMENT_64
  cmdsize 72
  segname __LINKEDIT
   vmaddr 0x00000001026dc000
   vmsize 0x000000000013aaf8
  fileoff 40321024
 filesize 1288952
  maxprot 0x00000001
 initprot 0x00000001
   nsects 0
    flags 0x0
Load command 4
            cmd LC_DYLD_INFO_ONLY
        cmdsize 48
     rebase_off 40321024
    rebase_size 22784
       bind_off 40343808
      bind_size 41008
  weak_bind_off 40384816
 weak_bind_size 128
  lazy_bind_off 40384944
 lazy_bind_size 43928
     export_off 40428872
    export_size 767840
Load command 5
     cmd LC_SYMTAB
 cmdsize 24
  symoff 41498624
   nsyms 1698
  stroff 41540584
 strsize 69392
Load command 6
            cmd LC_DYSYMTAB
        cmdsize 80
      ilocalsym 0
      nlocalsym 1
     iextdefsym 1
     nextdefsym 1
      iundefsym 2
      nundefsym 1696
         tocoff 0
           ntoc 0
      modtaboff 0
        nmodtab 0
   extrefsymoff 0
    nextrefsyms 0
 indirectsymoff 41525792
  nindirectsyms 3697
      extreloff 0
        nextrel 0
      locreloff 0
        nlocrel 0
Load command 7
          cmd LC_LOAD_DYLINKER
      cmdsize 32
         name /usr/lib/dyld (offset 12)
Load command 8
     cmd LC_UUID
 cmdsize 24
    uuid 80B206C4-27D3-3B06-A4BA-E8E0BB5D8AE2
Load command 9
      cmd LC_VERSION_MIN_MACOSX
  cmdsize 16
  version 10.12
      sdk 12.1
Load command 10
      cmd LC_SOURCE_VERSION
  cmdsize 16
  version 0.0
Load command 11
       cmd LC_MAIN
   cmdsize 24
  entryoff 21984
 stacksize 0
Load command 12
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1856.105.0
compatibility version 300.0.0
Load command 13
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libobjc.A.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 228.0.0
compatibility version 1.0.0
Load command 14
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libSystem.B.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1311.0.0
compatibility version 1.0.0
Load command 15
          cmd LC_LOAD_DYLIB
      cmdsize 88
         name /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 2113.20.111
compatibility version 45.0.0
Load command 16
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 88
         name /System/Library/Frameworks/Combine.framework/Versions/A/Combine (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 278.102.0
compatibility version 1.0.0
Load command 17
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1856.105.0
compatibility version 150.0.0
Load command 18
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1141.1.0
compatibility version 1.0.0
Load command 19
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/CryptoKit.framework/Versions/A/CryptoKit (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.0.0
compatibility version 1.0.0
Load command 20
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/JavaScriptCore.framework/Versions/A/JavaScriptCore (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 612.3.6
compatibility version 1.0.0
Load command 21
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name /usr/lib/libc++.1.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1200.3.0
compatibility version 1.0.0
Load command 22
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libncurses.5.4.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 5.4.0
compatibility version 5.4.0
Load command 23
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name /usr/lib/libz.1.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.2.11
compatibility version 1.0.0
Load command 24
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftAppKit.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 109.0.0
compatibility version 1.0.0
Load command 25
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name @rpath/libswiftCore.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1300.0.46
compatibility version 1.0.0
Load command 26
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftCoreData.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 19.0.0
compatibility version 1.0.0
Load command 27
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name @rpath/libswiftCoreFoundation.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 14.0.0
compatibility version 1.0.0
Load command 28
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name @rpath/libswiftCoreGraphics.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 2.0.0
compatibility version 1.0.0
Load command 29
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftCoreImage.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 2.0.0
compatibility version 1.0.0
Load command 30
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name @rpath/libswiftDarwin.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 0.0.0
compatibility version 1.0.0
Load command 31
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name @rpath/libswiftDispatch.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 11.0.0
compatibility version 1.0.0
Load command 32
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name @rpath/libswiftFoundation.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 70.101.0
compatibility version 1.0.0
Load command 33
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftIOKit.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.0.0
compatibility version 1.0.0
Load command 34
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftMetal.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 258.17.0
compatibility version 1.0.0
Load command 35
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name @rpath/libswiftObjectiveC.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 3.0.0
compatibility version 1.0.0
Load command 36
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftQuartzCore.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 3.0.0
compatibility version 1.0.0
Load command 37
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 56
         name @rpath/libswiftXPC.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.1.0
compatibility version 1.0.0
Load command 38
          cmd LC_RPATH
      cmdsize 32
         path /usr/lib/swift (offset 12)
Load command 39
          cmd LC_RPATH
      cmdsize 40
         path @executable_path/../lib (offset 12)
Load command 40
      cmd LC_FUNCTION_STARTS
  cmdsize 16
  dataoff 41196712
 datasize 284712
Load command 41
      cmd LC_DATA_IN_CODE
  cmdsize 16
  dataoff 41481424
 datasize 17200
cli/bin/sourcery (architecture arm64):
Load command 0
      cmd LC_SEGMENT_64
  cmdsize 72
  segname __PAGEZERO
   vmaddr 0x0000000000000000
   vmsize 0x0000000100000000
  fileoff 0
 filesize 0
  maxprot 0x00000000
 initprot 0x00000000
   nsects 0
    flags 0x0
Load command 1
      cmd LC_SEGMENT_64
  cmdsize 1832
  segname __TEXT
   vmaddr 0x0000000100000000
   vmsize 0x0000000002134000
  fileoff 0
 filesize 34816000
  maxprot 0x00000005
 initprot 0x00000005
   nsects 22
    flags 0x0
Section
  sectname __text
   segname __TEXT
      addr 0x0000000100001d60
      size 0x0000000001d58cc4
    offset 7520
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x80000400
 reserved1 0
 reserved2 0
Section
  sectname __stubs
   segname __TEXT
      addr 0x0000000101d5aa24
      size 0x0000000000002a18
    offset 30779940
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x80000408
 reserved1 0 (index into indirect symbol table)
 reserved2 12 (size of stubs)
Section
  sectname __stub_helper
   segname __TEXT
      addr 0x0000000101d5d43c
      size 0x00000000000029f4
    offset 30790716
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x80000400
 reserved1 0
 reserved2 0
Section
  sectname __objc_methlist
   segname __TEXT
      addr 0x0000000101d5fe30
      size 0x0000000000002530
    offset 30801456
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __const
   segname __TEXT
      addr 0x0000000101d62360
      size 0x000000000016c3c4
    offset 30810976
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_entry
   segname __TEXT
      addr 0x0000000101ece724
      size 0x0000000000000004
    offset 32302884
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __cstring
   segname __TEXT
      addr 0x0000000101ece730
      size 0x00000000001fc4c3
    offset 32302896
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __objc_methname
   segname __TEXT
      addr 0x00000001020cabf3
      size 0x0000000000002a30
    offset 34384883
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __swift5_typeref
   segname __TEXT
      addr 0x00000001020cd624
      size 0x000000000000b84c
    offset 34395684
     align 2^1 (2)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_reflstr
   segname __TEXT
      addr 0x00000001020d8e70
      size 0x0000000000008735
    offset 34442864
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_fieldmd
   segname __TEXT
      addr 0x00000001020e15a8
      size 0x000000000000cbe4
    offset 34477480
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_proto
   segname __TEXT
      addr 0x00000001020ee18c
      size 0x0000000000003618
    offset 34529676
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_types
   segname __TEXT
      addr 0x00000001020f17a4
      size 0x0000000000000fac
    offset 34543524
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_assocty
   segname __TEXT
      addr 0x00000001020f2750
      size 0x0000000000002970
    offset 34547536
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_builtin
   segname __TEXT
      addr 0x00000001020f50c0
      size 0x0000000000000708
    offset 34558144
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_protos
   segname __TEXT
      addr 0x00000001020f57c8
      size 0x00000000000000ec
    offset 34559944
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __swift5_capture
   segname __TEXT
      addr 0x00000001020f58b4
      size 0x0000000000007284
    offset 34560180
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __gcc_except_tab
   segname __TEXT
      addr 0x00000001020fcb38
      size 0x0000000000000044
    offset 34589496
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_classname
   segname __TEXT
      addr 0x00000001020fcb7c
      size 0x000000000000000e
    offset 34589564
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __objc_methtype
   segname __TEXT
      addr 0x00000001020fcb8a
      size 0x000000000000001f
    offset 34589578
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000002
 reserved1 0
 reserved2 0
Section
  sectname __unwind_info
   segname __TEXT
      addr 0x00000001020fcbac
      size 0x0000000000015404
    offset 34589612
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __eh_frame
   segname __TEXT
      addr 0x0000000102111fb0
      size 0x0000000000022044
    offset 34676656
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Load command 2
      cmd LC_SEGMENT_64
  cmdsize 632
  segname __DATA_CONST
   vmaddr 0x0000000102134000
   vmsize 0x00000000000b0000
  fileoff 34816000
 filesize 720896
  maxprot 0x00000003
 initprot 0x00000003
   nsects 7
    flags 0x10
Section
  sectname __got
   segname __DATA_CONST
      addr 0x0000000102134000
      size 0x00000000000017b0
    offset 34816000
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000006
 reserved1 898 (index into indirect symbol table)
 reserved2 0
Section
  sectname __mod_init_func
   segname __DATA_CONST
      addr 0x00000001021357b0
      size 0x00000000000000e0
    offset 34822064
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000009
 reserved1 0
 reserved2 0
Section
  sectname __const
   segname __DATA_CONST
      addr 0x0000000102135890
      size 0x00000000000ac148
    offset 34822288
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_classlist
   segname __DATA_CONST
      addr 0x00000001021e19d8
      size 0x0000000000000650
    offset 35527128
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_catlist
   segname __DATA_CONST
      addr 0x00000001021e2028
      size 0x0000000000000010
    offset 35528744
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_protolist
   segname __DATA_CONST
      addr 0x00000001021e2038
      size 0x0000000000000118
    offset 35528760
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_imageinfo
   segname __DATA_CONST
      addr 0x00000001021e2150
      size 0x0000000000000008
    offset 35529040
     align 2^2 (4)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Load command 3
      cmd LC_SEGMENT_64
  cmdsize 952
  segname __DATA
   vmaddr 0x00000001021e4000
   vmsize 0x00000000000b4000
  fileoff 35536896
 filesize 311296
  maxprot 0x00000003
 initprot 0x00000003
   nsects 11
    flags 0x0
Section
  sectname __la_symbol_ptr
   segname __DATA
      addr 0x00000001021e4000
      size 0x0000000000001c10
    offset 35536896
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000007
 reserved1 1656 (index into indirect symbol table)
 reserved2 0
Section
  sectname __objc_const
   segname __DATA
      addr 0x00000001021e5c10
      size 0x0000000000012950
    offset 35544080
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_selrefs
   segname __DATA
      addr 0x00000001021f8560
      size 0x0000000000000dc0
    offset 35620192
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000005
 reserved1 0
 reserved2 0
Section
  sectname __objc_protorefs
   segname __DATA
      addr 0x00000001021f9320
      size 0x0000000000000118
    offset 35623712
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_classrefs
   segname __DATA
      addr 0x00000001021f9438
      size 0x0000000000000118
    offset 35623992
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x10000000
 reserved1 0
 reserved2 0
Section
  sectname __objc_data
   segname __DATA
      addr 0x00000001021f9550
      size 0x0000000000007308
    offset 35624272
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __data
   segname __DATA
      addr 0x0000000102200858
      size 0x000000000002f229
    offset 35653720
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
Section
  sectname __thread_vars
   segname __DATA
      addr 0x000000010222fa88
      size 0x0000000000000060
    offset 35846792
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000013
 reserved1 0
 reserved2 0
Section
  sectname __thread_bss
   segname __DATA
      addr 0x000000010222fae8
      size 0x0000000000000020
    offset 0
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000012
 reserved1 0
 reserved2 0
Section
  sectname __bss
   segname __DATA
      addr 0x000000010222fb10
      size 0x00000000000671d8
    offset 0
     align 2^4 (16)
    reloff 0
    nreloc 0
     flags 0x00000001
 reserved1 0
 reserved2 0
Section
  sectname __common
   segname __DATA
      addr 0x0000000102296ce8
      size 0x00000000000006d9
    offset 0
     align 2^3 (8)
    reloff 0
    nreloc 0
     flags 0x00000001
 reserved1 0
 reserved2 0
Load command 4
      cmd LC_SEGMENT_64
  cmdsize 72
  segname __LINKEDIT
   vmaddr 0x0000000102298000
   vmsize 0x0000000000180000
  fileoff 35848192
 filesize 1564936
  maxprot 0x00000001
 initprot 0x00000001
   nsects 0
    flags 0x0
Load command 5
            cmd LC_DYLD_INFO_ONLY
        cmdsize 48
     rebase_off 35848192
    rebase_size 22944
       bind_off 35871136
      bind_size 49288
  weak_bind_off 35920424
 weak_bind_size 128
  lazy_bind_off 35920552
 lazy_bind_size 42728
     export_off 35963280
    export_size 766264
Load command 6
     cmd LC_SYMTAB
 cmdsize 24
  symoff 37016968
   nsyms 1677
  stroff 37054016
 strsize 68944
Load command 7
            cmd LC_DYSYMTAB
        cmdsize 80
      ilocalsym 0
      nlocalsym 1
     iextdefsym 1
     nextdefsym 1
      iundefsym 2
      nundefsym 1675
         tocoff 0
           ntoc 0
      modtaboff 0
        nmodtab 0
   extrefsymoff 0
    nextrefsyms 0
 indirectsymoff 37043800
  nindirectsyms 2554
      extreloff 0
        nextrel 0
      locreloff 0
        nlocrel 0
Load command 8
          cmd LC_LOAD_DYLINKER
      cmdsize 32
         name /usr/lib/dyld (offset 12)
Load command 9
     cmd LC_UUID
 cmdsize 24
    uuid A24D87E0-5C7D-30DF-80D0-287751C9A700
Load command 10
      cmd LC_BUILD_VERSION
  cmdsize 32
 platform 1
    minos 11.0
      sdk 12.1
   ntools 1
     tool 3
  version 711.0
Load command 11
      cmd LC_SOURCE_VERSION
  cmdsize 16
  version 0.0
Load command 12
       cmd LC_MAIN
   cmdsize 24
  entryoff 7520
 stacksize 0
Load command 13
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1856.105.0
compatibility version 300.0.0
Load command 14
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libobjc.A.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 228.0.0
compatibility version 1.0.0
Load command 15
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libSystem.B.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1311.0.0
compatibility version 1.0.0
Load command 16
          cmd LC_LOAD_DYLIB
      cmdsize 88
         name /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 2113.20.111
compatibility version 45.0.0
Load command 17
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 88
         name /System/Library/Frameworks/Combine.framework/Versions/A/Combine (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 278.102.0
compatibility version 1.0.0
Load command 18
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1856.105.0
compatibility version 150.0.0
Load command 19
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1141.1.0
compatibility version 1.0.0
Load command 20
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/CryptoKit.framework/Versions/A/CryptoKit (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.0.0
compatibility version 1.0.0
Load command 21
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/JavaScriptCore.framework/Versions/A/JavaScriptCore (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 612.3.6
compatibility version 1.0.0
Load command 22
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name /usr/lib/libc++.1.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1200.3.0
compatibility version 1.0.0
Load command 23
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libncurses.5.4.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 5.4.0
compatibility version 5.4.0
Load command 24
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name /usr/lib/libz.1.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.2.11
compatibility version 1.0.0
Load command 25
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftAppKit.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 109.0.0
compatibility version 1.0.0
Load command 26
          cmd LC_LOAD_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftCore.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1300.0.46
compatibility version 1.0.0
Load command 27
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftCoreData.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 19.0.0
compatibility version 1.0.0
Load command 28
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 72
         name /usr/lib/swift/libswiftCoreFoundation.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 14.0.0
compatibility version 1.0.0
Load command 29
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 72
         name /usr/lib/swift/libswiftCoreGraphics.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 2.0.0
compatibility version 1.0.0
Load command 30
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftCoreImage.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 2.0.0
compatibility version 1.0.0
Load command 31
          cmd LC_LOAD_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftDarwin.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 0.0.0
compatibility version 1.0.0
Load command 32
          cmd LC_LOAD_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftDispatch.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 11.0.0
compatibility version 1.0.0
Load command 33
          cmd LC_LOAD_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftFoundation.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 70.101.0
compatibility version 1.0.0
Load command 34
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftIOKit.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.0.0
compatibility version 1.0.0
Load command 35
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftMetal.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 258.17.0
compatibility version 1.0.0
Load command 36
          cmd LC_LOAD_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftObjectiveC.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 3.0.0
compatibility version 1.0.0
Load command 37
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftQuartzCore.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 3.0.0
compatibility version 1.0.0
Load command 38
          cmd LC_LOAD_WEAK_DYLIB
      cmdsize 64
         name /usr/lib/swift/libswiftXPC.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 1.1.0
compatibility version 1.0.0
Load command 39
          cmd LC_RPATH
      cmdsize 32
         path /usr/lib/swift (offset 12)
Load command 40
          cmd LC_RPATH
      cmdsize 40
         path @executable_path/../lib (offset 12)
Load command 41
      cmd LC_FUNCTION_STARTS
  cmdsize 16
  dataoff 36729544
 datasize 287424
Load command 42
      cmd LC_DATA_IN_CODE
  cmdsize 16
  dataoff 37016968
 datasize 0
Load command 43
      cmd LC_CODE_SIGNATURE
  cmdsize 16
  dataoff 37122960
 datasize 290168
```

</details>

Reply:
Actually, I noticed this:

```
Load command 40
          cmd LC_RPATH
      cmdsize 40
         path @executable_path/../lib (offset 12)
```

Looks like `xcodebuild` is adding it but I'm not sure why.. I will add this:

```ruby
install_name_tool -delete_rpath "@executable_path/../lib" "#{CLI_DIR}bin/sourcery"`
```

Unless there is a better way to do strip it? 

Although that being said, it didn't appear to do any harm

Reply:
Done 21df06598e45c620fc11c98eb9f9639a085534fb